jdk-recent-unit-tests/src/test/java/com/uber/nullaway/jdk17/NullAwaySwitchTests.java

```diff
@@ -174,4 +175,30 @@ public void testSwitchExprLambda() {
             "}")
         .doTest();
   }
+
+  @Ignore
+  @Test
+  public void testSwitchExprNullCase() {
+    defaultCompilationHelper
+        .addSourceLines(
+            "SwitchExpr.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "class SwitchExpr {",
+            "  public enum NullableEnum {",
+            "    a,",
+            "    b,",
+            "  }",
+            "  // NOTE: we should report a bug here for nullableEnum but cannot do so until",
```

Comment:
Actually thinking more this is not true; if there is `case null` then there is no NPE. Can you confirm?

Reply:
I'm saying that NullAway should _not_ report an NPE here, because there will be no NPE if `case null` is present.  See [here](https://stackoverflow.com/a/10332167).  You only get an NPE if there is no `case null` case.

Reply:
That's correct, I misunderstood your comment. I added the comment for NullAway not reporting without null, as we discussed previously, but in this context  i can either omit it entirely or mention that it only applies when null case isn't defined.

Reply:
Yeah, I misunderstood before.  I went ahead and deleted the comment.